admin/acceptance.py

```diff
@@ -106,6 +106,13 @@ def start_nodes(self):
             cwd=self.vagrant_path.path,
             env=extend_environ(FLOCKER_BOX_VERSION=box_version))
 
+        from flocker.provision._install import run, task_pull_docker_images
+        for node in self.NODE_ADDRESSES:
+            run(
+                username='root',
+                address=node,
+                commands=task_pull_docker_images()
```

Comment:
I'm fine with this differeing marginally from docs on presumption that a green buildbot is more important than testing purity, and that Docker doing implicit pulls isn't going to stop happening anytime soon. Increasing timeouts is pretty terrible, we want tests to fail quickly.